File: Arcmage.Web/src/app/models/deck-search-options.ts

import { SearchOptionsBase } from "./search-options-base";
import { Status } from "./status";
import { Language } from "./language";

export class DeckSearchOptions extends SearchOptionsBase {
   exportTiles: boolean | null;
   status: Status;
   myDecks: boolean | null;
   excludeDrafts: boolean | null;
   language: Language;
}

File: Arcmage.Model/DeckSearchOptions.cs
namespace Arcmage.Model
{
    public class DeckSearchOptions : SearchOptionsBase
    {
        public bool? ExportTiles { get; set; }

        public Status Status { get; set; }

        public Language Language { get; set; }

        public bool? MyDecks { get; set; }

        public bool? ExcludeDrafts { get; set; }
    }
}
